applications/DEMApplication/python_scripts/main_script.py

```diff
@@ -501,7 +501,7 @@ def SetInitialNodalValues(self):
     def InitializeTimeStep(self):
         pass
 
-    def BeforeSolveOperations(self, time):
+    def _BeforeSolveOperations(self, time):
```

Comment:
I think we could leave the old method calling the new one plus printing a big warning on screen (with expiration date), in case someone is still using it from private applications. After some time we completely remove the old version. This is what we are doing in the Kratos core for every api breaker. What do you think?

Reply:
Ok!

Reply:
I suppose we dont't need to ensure backward compatibility with other methods

Reply:
I would follow the same process with all methods that get deprecated. As systematically as possible.